mail/templates/product_order_receipt/body.html

```diff
@@ -0,0 +1,88 @@
+{% extends "email_base.html" %}
+
+{% block content %}
+<tr class="receipt-new-wrapper" style="font: 14px/22px 'Arial', sans-serif; margin: 0 auto; padding: 0 20px">
+    <table style="margin: 0 20px !important;">
+    <td class="rec-head" style="color: rgba(0,0,0,0.85); font: 14px/20px 'Arial', sans-serif; max-width: 696px; ">
+        <p style="font-weight: normal; margin: 0 0 20px;">Dear {{ purchaser.name }},</p>
+        <p style="font-weight: normal; margin: 0 0 20px;">You have been enrolled {% if content_title %} in {{ content_title }}{% endif %}.
+            The course should now appear on your MITxOnline <a href="{{ base_url }}{% url 'user-dashboard' %}" style="color: #0070DA">dashboard</a>. You can also access your receipt by <a style="color: #0070DA" href="{{ base_url }}/orders/receipt/{{order.id }}">clicking here</a>.
+        </p>
+        <p style="font-weight: normal; margin: 0 0 20px;">Below you will find a copy of you receipt:</p>
+    </td>
+    <tr>
+        <td>
+            <h1 style="color: #000000; font-size: 24px; font-weight: 700; line-height: 26px; margin: 0 0 24px;">MITxOnline</h1>
+            <h2 style="color: #000000; font-size: 22px; font-weight: 700; line-height: 24px; margin: 0 0 22px;">Receipt</h2>
+            <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px; margin: 0 0 20px;">Order Information</h3>
+            <p>
+                {% if order.reference_number  %}<strong style="font-weight: 700;">Order Number:</strong> {{ order.reference_number }}<br> {% endif %}
+                <strong style="font-weight: 700;">Order Date:</strong><span> {{ order.created_on|date:"F j, Y"}} </span><br>
+                <strong style="font-weight: 700;">Order Total:</strong> ${{ order_total }}<br>
+            </p>
+            {% for line in lines %}
+            <p>
+                <strong style="font-weight: 700;">Order Item:</strong> {{ line.content_title }}<br>
```

Comment:
Probably should include course number here as well. The names aren't very distinct.